src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    public function versionsTabAction(ContentView $view)
+    {
+        $repository = $this->getRepository();
+        $contentInfo = $view->getContent()->getVersionInfo()->getContentInfo();
+        $contentService = $repository->getContentService();
+        $versions = $contentService->loadVersions($contentInfo);
+
+        $authors = $this->loadUsersForVersions($repository, $versions);
```

Comment:
I don't think you need to create a local variable for this. You could move the call directly to the `addParameters` call below.